packages/cli/package.json

```diff
@@ -92,7 +92,7 @@
     "@vercel/fetch-retry": "5.0.3",
     "@vercel/frameworks": "0.7.1",
     "@vercel/ncc": "0.24.0",
-    "@vercel/nft": "0.17.5",
+    "@vercel/nft": "0.18.1",
```

Comment:
We're not actually using nft in CLI itself (I think this was leftover from the v2 vc build).

```suggestion
```

Reply:
Should the lock file change as well? I tried running `yarn install` from `vercel/packages/cli` with this change and its not changing the lock file

Reply:
I wouldn't expect it to change since nft is still being referenced by these other packages.